modules/programs/k9s.nix

```
@@ -8,9 +8,13 @@ let
   yamlFormat = pkgs.formats.yaml { };
 
 in {
-  meta.maintainers = with maintainers; [ katexochen liyangau ];
+  meta.maintainers = with maintainers; [
+    katexochen
+    liyangau
+    hm.maintainers.LucasWagler
+  ];
 
-  options.programs.k9s = {
+  options.programs.k9s = rec {
```

Shouldn't be necessary to use rec here:

```suggestion
  options.programs.k9s = {
```